server/server.go

```diff
@@ -2,13 +2,15 @@ package server
 
 import (
 	"fmt"
+	"os"
 
 	"github.com/mrunalp/ocid/oci"
 	"github.com/mrunalp/ocid/utils"
 )
 
 const (
 	runtimeAPIVersion = "v1alpha1"
+	imageStore        = "/var/lib/ocid/images"
```

Comment:
@runcom , @mrunalp , should we provide an option of `ocid` for setting this path, like `--sandboxDir` and `--containerDir`?